project_event/report/project_task_activity_report.xml

```diff
@@ -84,14 +73,14 @@
               <th>Department</th>
               <th>Employee</th>
               <th>Expected Arrival</th>
-              <th>Expected Departure</th>
+              <th>Real Arrival</th>
             </tr>
             <t t-foreach="tasks" t-as="task">
               <tr>
                 <td> <t t-esc="task['department']"/></td>
                 <td> <t t-esc="task['employee']"/></td>
                 <td> <t t-esc="datetime.datetime.strptime(task['expected_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
-                <td> <t t-esc="datetime.datetime.strptime(task['expected_departure'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
+                <td> <t t-esc="datetime.datetime.strptime(task['real_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
```

Comment:
real_start is not in tasks, it is in task_details

Reply:
Right, added to tasks